tools/update-resources.sh

```diff
@@ -4,20 +4,22 @@ getResource() {
   local TEMP_FILE="$(mktemp)"
   echo "$1"
   if curl --compressed --fail --progress-bar "$1" > "$TEMP_FILE"; then
```

Comment:
Would you change `--progress-bar` to `--silent` please? Otherwise looks good, thanks for speeding the process up!